lib/park.rb

```diff
@@ -0,0 +1,23 @@
+class Park
+  attr_reader :name, :admission_price, :vehicles
+  def initialize(park_details)
+    @name = park_details[:name]
+    @admission_price = park_details[:admission_price]
+    @vehicles = park_details[:vehicles] = []
+    @revenue = park_details[:revenue] = 0
```

Comment:
I'm not seeing hash keys for `revenue` or `vehicles` in your tests so I don't think you actually need these. You can set them equal to `[]` and `0` directly.
```suggestion
    @vehicles = []
    @revenue = 0
```